README.md

```diff
@@ -42,3 +42,26 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
 
 To learn React, check out the [React documentation](https://reactjs.org/).
+
+## Commit Message Convention
+
+This project enforces Conventional commits. See [Conventional Commits specification](https://www.conventionalcommits.org/).
+
+In short:
```

Comment:
you could add an example of how a commit message would be.
`docs[tooling]: added documentation on commit msg validation`

Reply:
fixed!